micrometer-core/src/main/java/io/micrometer/core/instrument/AbstractTimer.java

```diff
@@ -134,6 +136,17 @@ private void initPauseDetector(PauseDetector pauseDetectorType) {
         if (pauseDetectorType instanceof NoPauseDetector) {
             return;
         }
+        if (pauseDetectorType instanceof ClockDriftPauseDetector) {
+            try {
+                Class.forName("org.LatencyUtils.SimplePauseDetector");
+            }
+            catch (ClassNotFoundException e) {
+                pauseDetectorCNFELogger.log(
+                        "LatencyUtils is not on the runtime classpath but a ClockDriftPauseDetector which requires it is configured. Pause detection is disabled.",
```

Comment:
How about also explicitly telling the users what to do?
```suggestion
                        "LatencyUtils is not on the runtime classpath but a ClockDriftPauseDetector which requires it is configured. Pause detection is disabled, please add LatencyUtils to the classpath if you want to enable it.",
```